ndk/sample/build.gradle.kts

```diff
@@ -11,7 +11,7 @@ android {
 
     defaultConfig {
         applicationId = "io.sentry.ndk.sample"
-        minSdk = 19
+        minSdk = 21
```

Comment:
Bumped bc Android Studio was complaining about the old AGP version; seems fine to do this, since [sentry-Java also bumped 4 months ago](https://github.com/getsentry/sentry-java/pull/3852).

Reply:
Should be fine, as it's just the sample anyway.